pkg/executables/executables.go

```diff
@@ -167,3 +178,16 @@ func execute(ctx context.Context, cli string, in []byte, args ...string) (bytes.
 	}
 	return stdout, nil
 }
+
+func executeCleanup(ctx context.Context, containerName string) {
+	dockerCommands := []string{
+		"rm", "-f", "-v", containerName,
+	}
+	cmd := exec.CommandContext(ctx, "docker", dockerCommands...)
+	logger.V(6).Info("Executing clean up command", "cmd", cmd)
```

Comment:
Let's include name here, not sure if the format is as below
```suggestion
	logger.V(6).Info("cleaning up command", "cmd", cmd, "name", containerName)
```

Reply:
the command already shows the container name; for eg

`Executing clean up command	{"cmd": "/usr/local/bin/docker rm -f -v eksa_kubectl_2021-09-2813.25.27.128915"}`

Reply:
Let's change the text tho